po/de.po

```diff
@@ -1093,32 +1110,37 @@ msgid ""
 "tag hidden in the Actionable view, etc.). To avoid this, you may prefer to "
 "edit your task while in the &quot;Open&quot; tasks view instead."
 msgstr ""
-"Wenn Sie den &quot;Arbeitsansicht&quot; Knopf drücken, werden nur "
+"Wenn Sie den &quot;Bearbeitbar&quot;-Tab oben auswählen, werden nur "
 "bearbeitbare Aufgaben angezeigt.\n"
 "\n"
 "Was ist eine bearbeitbare Aufgabe? Es ist eine Aufgabe, die Sie jetzt direkt "
 "ausführen können.\n"
 "\n"
-"Es ist eine Aufgabe, die &quot;startbar&quot; ist, z.B. das Startdatum ist "
-"bereits vorbei.\n"
+"Es ist eine Aufgabe, die &quot;startbar&quot; ist, das heißt, das Startdatum "
+"ist bereits eingetroffen.\n"
+"\n"
+"Es ist eine Aufgabe ohne offene Teilaufgaben, also können Sie die Aufgabe "
+"direkt ausführen, weil es nicht mehr von einer Aufgabe abhängt.\n"
 "\n"
-"Es ist eine Aufgabe ohne offene Teilaufgaben, z.B. können Sie die Aufgabe "
-"direkt ausführen.\n"
+"Es ist eine Aufgabe, dessen Fälligkeitsdatum etwas anderes als &quot;"
+"Irgendwann&quot; ist, weil dieses spezielle Datum für Aufgaben da ist, die "
+"mehr Bedenkzeit benötigen, bevor Sie ausgeführt werden können.\n"
 "\n"
 "Die Arbeitsansicht wird Ihnen also nur Aufgaben zeigen, die Sie unmittelbar "
-"bearbeiten können.\n"
+"bearbeiten können, und ist deshalb nützlich, wenn Sie auf die nächste "
+"Aufgaben fokussieren wollen um etwas fertig zu bringen!\n"
 "\n"
 "Wenn Sie Schlagworte verwenden, können Sie auf ein Schlagwort in der "
-"Seitenleiste klicken und Aufgaben, die diesem Schlagwort zugeordnet sind, in "
-"der Arbeitsansicht verbergen. Das ist sehr sinnvoll, wenn Sie ein Schlagwort "
-"wie &quot;irgendwann&quot; für Aufgaben verwenden, die nicht besonders "
-"dringend sind, aber die Sie erledigen wollen."
+"Seitenleiste rechts-klicken und bearbeiten, und dann auswählen, ob Aufgaben "
+"mit dem Schlagwort in der Arbeitsansicht verborgen werden sollen oder nicht. "
+"Das ist sehr sinnvoll, wenn Sie ein Schlagwort wie &quot;warten&quot; für "
+"Aufgaben verwenden, die durch etwas Externes blockiert ist (z. B. ein "
```

Comment:
"sind" instead of "ist" for grammatical correctness because "Aufgaben" is plural.